assets/tuned/daemon/tuned/profiles/functions/function_set_pstate.py

```diff
@@ -0,0 +1,39 @@
+import os
```

Comment:
We no longer use the bundled tuned. The function has to be contributed to https://github.com/redhat-performance/tuned/

Reply:
Okay.